static/app/views/organizationGroupDetails/issueQuickTrace.tsx

```diff
@@ -0,0 +1,210 @@
+    );
+  }
+
+  snoozePrompt = () => {
+    const {api, event, organization} = this.props;
+    const data = {
+      projectId: event.projectID,
+      organizationId: organization.id,
+      feature: 'quick_trace_missing',
+      status: 'snoozed' as const,
+    };
+    promptsUpdate(api, data).then(() => this.setState({shouldShow: false}));
+  };
+
+  renderQuickTrace(results) {
+    const {event, location, organization} = this.props;
+    const {shouldShow} = this.state;
+    const {isLoading, error, trace, type} = results;
+
+    if (isLoading) {
+      return <Placeholder height="24px" />;
+    }
+
+    if (error || trace === null || trace.length === 0) {
+      if (!shouldShow) {
+        return null;
+      }
+
+      return (
+        <StyledAlert type="info" icon={<IconInfo size="sm" />}>
+          <AlertContent>
+            {tct('The [type] for this error cannot be found. To see why, [link].', {
+              type: type === 'missing' ? t('transaction') : t('trace'),
+              link: (
+                <ExternalLink href="https://docs.sentry.io/product/performance/trace-view/#troubleshooting">
+                  go to docs
+                </ExternalLink>
+              ),
+            })}
+            <Actions>
+              <Button
+                priority="link"
+                title={t('Dismiss for a month')}
+                onClick={this.snoozePrompt}
+              >
+                <IconClose />
+              </Button>
+            </Actions>
+          </AlertContent>
+        </StyledAlert>
+      );
+    }
+
+    return (
+      <QuickTrace
+        event={event}
+        quickTrace={results}
+        location={location}
+        organization={organization}
+        anchor="left"
+        errorDest="issue"
+        transactionDest="performance"
+      />
+    );
+  }
+
+  render() {
+    const {event, organization, location} = this.props;
+
+    return (
+      <ErrorBoundary mini>
+        <QuickTraceQuery event={event} location={location} orgSlug={organization.slug}>
+          {results => {
+            return (
+              <React.Fragment>
+                {this.renderTraceLink(results)}
+                <QuickTraceWrapper>{this.renderQuickTrace(results)}</QuickTraceWrapper>
+              </React.Fragment>
+            );
+          }}
+        </QuickTraceQuery>
+      </ErrorBoundary>
+    );
+  }
+}
+
+const LinkContainer = styled('span')`
+  margin-left: ${space(1)};
+  padding-left: ${space(1)};
+  position: relative;
+  font-weight: normal;
+
+  &:before {
+    display: block;
+    position: absolute;
+    content: '';
+    left: 0;
+    top: 2px;
+    height: 14px;
+    border-left: 1px solid ${p => p.theme.border};
+  }
+`;
+
+const QuickTraceWrapper = styled('div')`
+  margin-top: ${space(0.5)};
+`;
+
+const StyledAlert = styled(Alert)`
+  margin: 0;
+`;
+
+const AlertContent = styled('div')`
+  display: flex;
+  flex-wrap: wrap;
+
+  @media (min-width: ${p => p.theme.breakpoints[0]}) {
```

Comment:
👏  Nice breakpoint check

Reply:
Gotta give credit to @EvanPurkhiser for this one